### ISSUE TITLE
Issue running git clone to download the framework.

### ISSUE THREAD

**Issue by nickngqs** [foundation/foundation-emails#824]
**How can we reproduce this bug?**
```
foundation new --framework emails
```

**What did you expect to happen?**
Initialise the project

**What happened instead?**
```
Downloading the project template...There was an issue running git clone to download the framework Foundation for emails and template unspecified.
Make sure your computer's Git is configured properly and then try again.
```

**Additional Information**
node v6.9.2
nom v3.10.9

Try several suggested solutions both on Zurb forum and github issues. Doesn't seem to work at all. And the weird part is it work just last week.

Solutions tried
- Install xcode select
- Make sure command line is correct
- Make sure permissions of folder is correct
- Use standard git clone instead

I've tried removing all dependencies except for babel just for testing and it runs fine. Doesn't seem to have cloning issues.

**Comment by rafibomb** [comment]
Thanks for the report! This is the only report we've seen and it seems to work now. Maybe a temporary glitch? We'll close for now,  open a new issue if you can reproduce still.

**Comment by Wirelesstaco** [comment]
Just had the same issue.

Turns out it was a high sierra issue and I had to reinstall xcode

`xcode-select --install`

It was related to [this](https://apple.stackexchange.com/questions/254380/macos-sierra-invalid-active-developer-path)

**Comment by katlis** [comment]
Had this issue too on High Sierra. A `git status` told me to run `sudo xcodebuild -license` to agree to the new Xcode license. That did the trick.

**Comment by icreatesolutions** [comment]
I'm receiving this issue with Windows.

Node 8.11.3
NPM 6.2.0

EDIT: Fixed when updating git from 1.9.5 to 2.10.2

**Comment by zmarmara** [comment]
Hi, I'm just posting this to help Windows users who will receive the error message.
You have to launch Command Prompt as an administrator. Use the "win + x" keyboard shortcut to open "Command Prompt (Admin)" from the menu.
